databases/functions/userFollowUser/check-is-blurred-post.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * @param {Model} userFollowUserModel
+ * @param {RegisterBodyData.Users} users
+ */
+module.exports = async (userFollowUserModel, userIdFollower, _transaction = null) => {
+  let isBlurredPost = false;
+
+  const signUserList = await userFollowUserModel.findAll({
+    where: {
+      user_id_follower: userIdFollower,
+      is_anonymous: false
+    },
+    raw: true
+  });
```

Comment:
Mas @brm-stnd please add limit 7 on findAll.
So we don't have to query all of the user's following.

Reply:
baik mas